### PR TITLE
Feature/aboutpage

### PR DESCRIPTION
**Pull request by Leasw144**
# Turing Index Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
* Implemented routing feature for navigating to /map, /about and 404 instances
* Created 404 component under PageNotFound
* Basement link under menu now brings user to /map
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing (add-testing to project)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [x] No
- [ ] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`manually tested`
# Message for reviewer:
`Your message here`
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/app/heading/heading.component.html [hunk 1]
```diff
@@ -6,13 +6,14 @@
     <div class="navigation__background">&nbsp;</div>
     <nav class="navigation__nav">
         <ul class="navigation__list">
-            <li class="navigation__item"><a href="#" class="navigation__link"><span>01</span>Basement</a></li>
-            <li class="navigation__item"><a href="https://rickrolled.com/" class="navigation__link"><span>02</span>Community</a></li>
-            <li class="navigation__item"><a href="#" class="navigation__link"><span>03</span>About</a></li>
-            <li class="navigation__item"><a href="#" class="navigation__link"><span>04</span>placeholder</a></li>
-            <li class="navigation__item"><a href="#" class="navigation__link"><span>05</span>placeholder</a></li>
+            <li class="navigation__item"><a routerLink="/map" class="navigation__link"><span>01</span>Basement</a></li>
```
Comment (JoshSevy) [inline src/app/heading/heading.component.html:9]: This looks great! These routes are going to be extremely useful as the project scales.

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import { CommonModule } from '@angular/common';
+import { AboutComponent } from './about/about.component';
+import { MapComponent } from './map/map.component'
+import { PageNotFoundComponent } from './page-not-found/page-not-found.component'
+
+const routes: Routes = [
+  { path: '', redirectTo: '/map', pathMatch: 'full' },
+  { path: 'map', component: MapComponent },
+  { path: 'about', component: AboutComponent },
+  { path: '**', component: PageNotFoundComponent }
```
Comment (NicoleLatifi) [inline src/app/app-routing.module.ts:12]: Great job thinking about adding this error handling!

File: src/app/turing-map/turing-map.component.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<div>
+  <button (click)="onlick()">turing-map works! BUT WHERE THE FUCK ARE YOU</button>
```
Comment (JoshSevy) [inline src/app/turing-map/turing-map.component.html]: Space?

File: .Procfile [hunk 4]
```diff
@@ -0,0 +1 @@
+web: npm run start:prod
```
Comment (JoshSevy) [inline .Procfile:1]: hopefully this helps with the heroku deployment

**Review by NicoleLatifi — COMMENTED** [review]
Great job adding the about page and routing.

File: src/app/app.component.spec.ts [hunk 5]
```diff
@@ -16,16 +16,16 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'web-app'`, () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.componentInstance;
-    expect(app.title).toEqual('web-app');
-  });
+  // it(`should have as title 'web-app'`, () => {
```
Comment (JoshSevy) [inline src/app/app.component.spec.ts:19]: Once the app develops more we can modify these tests so we have a good amount of test coverage.

**Review by JoshSevy — APPROVED** [review]
Great work Team!

**Pull request merged**